utils/custom/validate.helpers.js

```diff
@@ -0,0 +1,58 @@
+const isValidDate = dateString => {
+  return !isNaN(Date.parse(dateString));
+}
+
+const validateStringFormat = (pattern, string) => {
+  var check = new RegExp(pattern);
+  var res = check.test(string);
+
+  return res;
+}
+
+const validateTelephoneFormat = string => {
+    // validates format: XXX-XXX-XXXX
+    const format = /^(\+0?1\s)?\(?\d{3}\)?[-]\d{3}[-]\d{4}$/
+    return validateStringFormat(format, string)
+}
+
+const validateDateFormat = string => {
+    // validates format: YYYY-MM-DD
+    const format = /(\d{4})-(\d{2})-(\d{2})/
```

Comment:
should this regex not capture the beginning and end of the string like the others so: 
`/^(\d{4})-(\d{2})-(\d{2})$/`

Reply:
If fixed this in my push